redaxo/src/core/lib/response.php

```diff
@@ -292,13 +292,14 @@ public static function sendContent($content, $contentType = null, $lastModified
             ) {
                 self::sendLastModified($lastModified);
             }
-
             // ----- ETAG
-            if (!self::$sentEtag
-                && (true === rex::getProperty('use_etag') || rex::getProperty('use_etag') === $environment)
-            ) {
-                self::sendEtag($etag ?: self::md5($content));
+            if (self::$sentEtag) {
+                return;
+            }
+            if (true !== rex::getProperty('use_etag') && rex::getProperty('use_etag') !== $environment) {
+                return;
```

Comment:
Die returns hier scheinen nicht korrekt zu sein, es muss unten mit dem Code ab zeile 305 weitergehen.

Reply:
danke - revert in https://github.com/redaxo/redaxo/pull/4137